Makefile

```diff
@@ -1,5 +1,5 @@
-.PHONY: build dist lint setup copy serve clean metrics test coverage zip contrib flow
-build: test build/katex.min.js build/katex.min.css contrib zip compress
+.PHONY: build dist lint setup webpack serve clean metrics test coverage zip flow
```

Comment:
Assuming most of the infrastructure related changes go away once the webpack PR is merged? 

What changes are specific to this PR?

Reply:
@rrandallcainc Currently only the last commit(https://github.com/Khan/KaTeX/pull/1074/commits/f3b64be8f5a6d2db03387a29e688e514c0a87373) is specific to this PR.

Reply:
Doh, should have read your full comment :)